dockerfiles/agent/Dockerfile

```diff
@@ -6,8 +6,15 @@ RUN rm -rf vendor
 RUN go mod download
 RUN CGO_ENABLED=0 go build -o ./build/agent agent/cmd/main.go
 
+RUN touch ./server.cert
+RUN touch ./server.key
+RUN chown 65532 ./server.cert
+RUN chown 65532 ./server.key
+
 FROM scratch
 COPY --from=builder ./build/agent agent
+COPY --from=builder ./server.cert server.cert
+COPY --from=builder ./server.key server.key
```

Comment:
why the ceritificates managed with container image ? 
certificate expected to be part of secrets. 
Secrets with cert generated with cert-manager

Reply:
While configuring certs to rest server, http.ListenServe expecting files as input
So I did that way.
Secret is already created. If secret is already present I am creating files still to configure rest server.

I am thinking one more option using empty dir mount so that it will be temporary mount till the pod is running. With this docker file crt and key files not required to create.
what do you think about this option?

Reply:
Incorporated the changes. Will test and confirm.